lib/react_client.dart

```diff
@@ -235,10 +244,7 @@ ReactComponentFactory _registerComponent(ComponentFactory componentFactory, [Ite
     )])
   ]);
 
-  /**
-   * return ReactComponentFactory which produce react component with set props and children[s]
-   */
-  return (Map props, [dynamic children]) {
+  var _call = (Map props, [dynamic children]) {
```

Comment:
(Minor) This is inside a closure, so prefixing the name with an underscore to make it private doesn't do anything.